From 5806fa61ae64b23b3b17b5c6656278e2d4caa8f7 Mon Sep 17 00:00:00 2001
From: Jianhui Zhao <zhaojh329@gmail.com>
Date: Fri, 26 Jul 2024 18:57:48 +0800
Subject: [PATCH] fix(nl80211): Fixed a variable name written incorrectly

This case a crash if there is an IBSS for scan.

Signed-off-by: Jianhui Zhao <zhaojh329@gmail.com>
---
 nl80211.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nl80211.lua b/nl80211.lua
index ba7b6c1..0223580 100644
--- a/nl80211.lua
+++ b/nl80211.lua
@@ -593,7 +593,7 @@ local function parse_bss(nest)
     end
 
     if caps & 1 << 1 > 0 then
-        info.cap['IBSS'] = true
+        info.caps['IBSS'] = true
     end
 
     if caps & 1 << 4 > 0 then
-- 
2.34.1

